cmake/projects/Boost/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -286,6 +288,65 @@ file(
     "${using_mpi}\n"
 )
 
+# *** handle requested python version if such is specified ***
+#
+hunter_parse_cmake_args_for_keyword(
+  CMAKE_ARGS "@HUNTER_Boost_CMAKE_ARGS@"
+  KEYWORD REQUESTED_PYTHON_VERSION
+  OUTPUT requested_python
+)
+# determine paths required to set up python in boost.user.jam
+if(requested_python)
+  #FindPythonInterp has been deprecated since CMake 3.12 in favor of
+  #FindPython, FindPython2, FindPython3
+  if(CMAKE_VERSION VERSION_LESS 3.12)
+    find_package(PythonInterp ${requested_python} EXACT QUIET)
+    if(PYTHONINTERP_FOUND)
+      set(python_executable ${PYTHON_EXECUTABLE})
+      execute_process(
+        COMMAND "${python_executable}" -c "import os.path, sysconfig; info = sysconfig.get_paths(); print(info['include'])"
+        RESULT_VARIABLE python_process
+        OUTPUT_VARIABLE python_include_directory
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+      )
+      if(NOT python_process EQUAL 0)
+        unset(python_include_directory)
+      endif()
+    endif()
+  else()
+    find_package(Python ${requested_python} EXACT QUIET COMPONENTS Interpreter Development)
+    if(Python_Interpreter_FOUND)
+      set(python_executable ${Python_EXECUTABLE})
+    endif()
+    if(Python_Development_FOUND)
+      list(GET Python_INCLUDE_DIRS 0 python_include_directory)
+    endif()
+  endif()
+  if(python_executable)
+    execute_process(
+      COMMAND "${python_executable}" -c "import os.path, sysconfig; info = sysconfig.get_paths(); print(os.path.dirname(info['stdlib']))"
+      RESULT_VARIABLE python_process
+      OUTPUT_VARIABLE python_library_directory
+      OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+    if(NOT python_process EQUAL 0)
+      unset(python_library_directory)
+    endif()
+  endif()
+
+  if(python_executable AND python_include_directory AND python_library_directory)
+    file(
+        APPEND ${boost_user_jam}
+        "using python  : ${requested_python} : \"${python_executable}\" : \"${python_include_directory}\" : \"${python_library_directory}\" ;\n"
+    )
+  else()
+    message(WARNING "Could not find some or all required components for the requested Python version ${requested_python}.")
```

Comment:
If user requested for Python support but some Python parts not found then it's fatal error. Assuming that Python should be installed by user (as a workaround for missing `hunter_add_package(python)`) then you can call `hunter_user_error` here.